react/ui-components/src/hoc/EditablePopup.js

```
@@ -0,0 +1,130 @@
+  //here form the formComposer config with column config and render inside children prop of Popup
+  
+  return (
+    <PopUp
+      className={'popUpClass'}
+      heading={t(`EDIT_ROW`)}
+      equalWidthButtons={true}
+      onOverlayClick={() => {
+        setShowEditablePopup(false);
+        setEditRow(null);
+      }}
+      children={[
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)